src/test/java/redis/clients/jedis/commands/jedis/HashesCommandsTest.java

```diff
@@ -200,7 +200,7 @@ public void hexists() {
 
   @Test
   public void hdel() {
-    Map<String, String> hash = new HashMap<String, String>();
```

Comment:
Do we need to consider the jdk 7 version? @sazzad16

Reply:
@zeekling No.